Lib/test/test_typing.py

```diff
@@ -10241,6 +10241,12 @@ def test_constructor(self):
     def test_repr(self):
         self.assertEqual(repr(NoDefault), 'typing.NoDefault')
 
+    def test_doc(self):
+        self.assertIsInstance(NoDefault.__doc__, str)
```

Comment:
This is not correct, `./python.exe -OO` will strip this. Please, add `@support.requires_docstrings`